docs/content/docs/getting-started/setup.md

```diff
@@ -33,25 +33,40 @@ You also need to build the core SDK once first:
 
 ## Building for SDK
 
-Whenever the native SDK changed, you need to (re)build the artifacts. To do that you can use `cargo make android`
+Whenever the native SDK changed, you need to (re)build the artifacts. To do that you can use the one of following commands
+
+`cargo make android`
+
+`cargo make ios`
+
+`cargo make desktop`
 
 .Note: currently only android is fully supported. Plumbing for iOS is existing but not tested, Web, Linux, Mac, Windows and are platforms have not been configured yet.
 
 ## Running the App
 
-Once the SDK is rebuild, you can run the flutter as usual on your device or emulator per:
+Once the SDK is rebuilded, you can run the flutter as usual on your device or emulator per:
 
-F5 in VS Code or `flutter run` in `app`
+F5 in VS Code or `flutter run` in `app` directory
+
+If you want to run flutter as desktop in windows:
+
+`flutter run -d windows`
```

Comment:
for me, if I don't choose a different emulator in VScode it does run the desktop version with `F5 Build` by default. does it not for you?

Reply:
`F5` depends on what was this option.
![image](https://user-images.githubusercontent.com/48786056/196813790-71c14fa7-3527-4635-b514-894f77c917f1.png)